core/templates/dev/head/pages/library/Library.js

```diff
@@ -65,6 +65,16 @@ oppia.controller('Library', [
     $scope.pageMode = LIBRARY_PATHS_TO_MODES[currentPath];
     $scope.LIBRARY_PAGE_MODES = LIBRARY_PAGE_MODES;
 
+    // Set page title
+    var title = '';
```

Comment:
Make `'Exploration Library - Oppia'` a default value for the title and then change it just when `$scope.pageMode === LIBRARY_PAGE_MODES.GROUP` or `$scope.pageMode === LIBRARY_PAGE_MODES.SEARCH`.